package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@jothepro/doxygen-awesome-css",
+  "version": "2.2.1-pre",
+  "description": "Custom CSS theme for doxygen html-documentation with lots of customization parameters.",
+  "main": "",
+  "scripts": {
+    "npm-pack": "npm pack",
+    "postversion": "git push origin --all && git push origin --tags",
```

Comment:
What happens if this is run on Windows?

Reply:
The same as if you run this on macOS or Linux. If you have git installed, it works as expected, otherwise you get an error.

Reply:
I mean, Windows doesn't ship with a POSIX shell, and here you seem to be passing a command to be invoked with `sh -c` . I don't use Windows, but I believe that this might cause issues there.

Reply:
I don't use Windows either, but the syntax seems valid:

```

C:\Users\ilg>git --version && git --version
git version 2.35.1.windows.2
git version 2.35.1.windows.2

C:\Users\ilg>
```

BTW, the Windows Git distribution comes with a terminal and bash.

Reply:
I'd opt for removing this script altogether. I am still evaluating to publish on npm, but if I decided to do so I would automate publishing in a github workflow triggered by the creation of the release tag, so the tag-pushing is not needed IMO.

Reply:
I removed it, but please note that when you use the `npm version` command, it also creates a tag, and exactly this tag should be pushed to the repo, otherwise you get into various troubles.

This is the reason in my projects I automated creating new versions with pushing the tag.

If you run into such issues, you can always add this line back.